governance/votes/Executive vote - August 22, 2024.md

```diff
@@ -0,0 +1,142 @@
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Update PSM State Variables in Conduit Contracts
+
+- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
+- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)
+
+If this executive proposal passes, then the PSM state variables in the following conduit contracts will be updated to point to [MCD_LITE_PSM_USDC_A](https://etherscan.io/address/0xf6e72db5454dd049d0788e411b06cfaf16853042):
+
+- [RWA014_A_INPUT_CONDUIT_URN](https://etherscan.io/address/0x6b86ba08bd7796464cea758061ac173d0268cf49)
+- [RWA014_A_INPUT_CONDUIT_JAR](https://etherscan.io/address/0x391470cd3d8307adc051d878a95fa9459f800dbc)
+- [RWA014_A_OUTPUT_CONDUIT](https://etherscan.io/address/0xd7cbdfde553de2063caafbf230be135e5dbb5064)
+- [RWA007_A_JAR_INPUT_CONDUIT](https://etherscan.io/address/0xc8bb4e2b249703640e89265e2ae7c9d5ea2af742)
+- [RWA007_A_INPUT_CONDUIT](https://etherscan.io/address/0x58f5e979ef74b60a9e5f955553ab8e0e65ba89c9)
+- [RWA007_A_OUTPUT_CONDUIT](https://etherscan.io/address/0x701c3a384c613157bf473152844f368f2d6ef191)
+- [RWA015_A_INPUT_CONDUIT_JAR_USDC](https://etherscan.io/address/0xb9373c557f3ae8cddd068c1644ed226cfb18a997)
+- [RWA015_A_INPUT_CONDUIT_URN_USDC](https://etherscan.io/address/0xe08cb5e24862ea86328295d5e5c08972203c20d8)
+- [RWA015_A_OUTPUT_CONDUIT](https://etherscan.io/address/0x1e86cb085f249772f7e7443631a87c6bdba2aceb)
+- [RWA009_A_INPUT_CONDUIT_URN_USDC](https://etherscan.io/address/0x08012ec53a7fabf6f33318dfb93c1289886ebbe1)
+
+### Migrate Reserves from PSM-USDC-A to LITE-PSM-USDC-A
+
+- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
+- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)
+
+If this executive proposal passes, **all but 200 million USDC reserves** from PSM-USDC-A will be migrated to LITE-PSM-USDC-A.
+
+### Update PSM-USDC-A Fees
+
+- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
+- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)
+
+If this executive proposal passes, then PSM-USDC-A fees will be updated as follows:
```

Comment:
```suggestion
If this executive proposal passes, then the PSM-USDC-A fees will be updated as follows:
```

Reply:
see above

Reply:
"the" refers to the fees in this sentence.

Reply:
Understood